src/hooks/useGoogleTagManager.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from "react"
+
+export const useGoogleTagManager = (gtmId?: string) => {
```

Comment:
Non-blocking:

Btw there is an actual [gatsby plugin for google tag manager](https://www.gatsbyjs.com/plugins/gatsby-plugin-google-tagmanager/), but I don't think we would benefit implementing it since it works wonderful with our current implementation.

Reply:
There are ready made custom hooks available for React as well and I was curious why you decided to write it yourself for Token Dashboard implementation. I just copy-pasted your solution with minor adjustments/improvements. If I didn't have your solution as a template I'd go for ready made hook.